README.md

```diff
@@ -1 +1,73 @@
-# equalizer
\ No newline at end of file
+# Wolox Equalizer
+[![circle-ci](https://img.shields.io/circleci/project/github/Wolox/wolox-equalizer.svg)](https://circleci.com/gh/Wolox/wolox-equalizer)
+[![npm](https://img.shields.io/npm/v/wolox-equalizer.svg)](https://www.npmjs.com/package/wolox-equalizer)
+
+[![FEArmy](./assets/FEA_icon.png)](https://github.com/orgs/Wolox/teams/front-end-army/members)
+## Features
+
+- Reset the default styles of the browser.
+- Contains our Wolox standards for normalizing front-end projects.
+
+## Installation
+
+#### npm
+```bash
+npm install --save wolox-equalizer
+```
+
+#### yarn
+```bash
+yarn add wolox-equalizer
+```
+
+## Usage
+
+1- Include the equalizer.scss to the root js file
```

Comment:
I would add **before the index file of your CSS/SCSS**

Reply:
In that case the developer will have to make something like `@import "../../node_modules/wolox-equalizer/equalizer" ` until we find a way to make an alias